app/models/page_element.rb

```diff
@@ -1,9 +1,15 @@
 # encoding: UTF-8
 class PageElement < ActiveRecord::Base
-   belongs_to :page
-   has_attached_file :picture, 
-                    :styles => { original: "4000x4000>", large: "1500x1500>", small: "250x250>",thumb: "100x100>" },                     
-                    :path => ":rails_root/public/system/images/sidor/element/:id/:style/:filename",
-                    :url => "/system/images/sidor/element/:id/:style/:filename"
-  validates_attachment_content_type :picture, :content_type => /\Aimage\/.*\Z/
+  belongs_to :page
+  has_attached_file :picture,
+                    styles: {original: "4000x4000>", large: "1500x1500>", small: "250x250>", thumb: "100x100>"},
```

Comment:
Line is too long. [112/100]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.